examples/complete/main.tf

```diff
@@ -0,0 +1,61 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 5.0.0"
+
+  name                 = local.vpc_name
+  cidr                 = var.vpc_cidr
+  private_subnets      = var.private_subnets
+  azs                  = var.availability_zones
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = {
+    Name        = "example-vpc"
+    Environment = "dev"
+  }
+}
+
+module "ecs_example" {
+  source = "../.."
+
+  logical_product_family  = "terratest"
+  logical_product_service = "ecs_appmesh"
+  class_env               = "dev"
+  instance_env            = 1
+  instance_resource       = 1
+  region                  = "us-east-1"
+
+  vpc_id          = module.vpc.vpc_id
+  private_subnets = module.vpc.private_subnets
+
+  namespace_name  = "<namespace_name>"
```

Comment:
there can be no placeholders in an example. Must be either static values or pointing to outputs of other modules. this comment it applicable to all such occurances in this file